bodhi/client/__init__.py

```diff
@@ -109,8 +109,8 @@ def _set_logging_debug(ctx, param, value):
     click.option('--request', help='Requested repository',
                  type=click.Choice(['testing', 'stable', 'unpush', 'batched'])),
     click.option('--autokarma', is_flag=True, help='Enable karma automatism'),
-    click.option('--stable-karma', type=click.INT, help='Stable karma threshold'),
-    click.option('--unstable-karma', type=click.INT, help='Unstable karma threshold'),
+    click.option('--stable-karma', type=click.INT, help='Stable karma threshold', default=3),
+    click.option('--unstable-karma', type=click.INT, help='Unstable karma threshold', default=-3),
```

Comment:
I think it's better to enforce the defaults server-side like we were doing before.